devicon.json

```
@@ -1002,6 +1002,25 @@
         "color": "#00d1b2",
         "aliases": []
     },
+    {
+        "name": "bun",
+        "altnames": [],
+        "tags": [
+            "javascript",
+            "zig",
+            "language"
+        ],
+        "versions": {
+            "svg": [
+                "original"
+            ],
+            "font": [
+                "original"
+            ]
```

We only allow mono-colored icons as fonts. 
For more info check out [SVG-Standards](https://github.com/devicons/devicon/wiki/SVG-Standards) and [SVG-Versions](https://github.com/devicons/devicon/wiki/SVG-Versions) in our wiki :)
```suggestion
            "font": []
```